docs/_templates/changelog/v3.0.0.md

```diff
@@ -24,6 +24,12 @@
 
 - <abbr title="New Features">✨</abbr> **[#279](https://github.com/mcbookshelf/Bookshelf/issues/279)** - Added a `#bs.block:play_block_sound` tag for playing block sounds.
 
+### `🌱 bs.generate`
+
+- <abbr title="Breaking Changes">⚠️</abbr> **[#296](https://github.com/mcbookshelf/Bookshelf/issues/296)** Renamed the `bs.generate` module to `bs.generation`.
```

Comment:
just an idea, but maybe we can put this at the start of the changelog to not be referenced for a single module but for the whole lib. But i'm fine with both choices

Reply:
I hesitated, but for me keeping that in the module section is better.